Docker_files/ocsci_container/scripts/edit_yaml.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import yaml
+import sys
+import os
+
+from pathlib import Path
+
+cli_args = sys.argv[1:]
+
+
+def send_cmd(cmd=None):
+    return subprocess.check_output(cmd, shell=True, universal_newlines=True)
+
+
+def yaml_to_dict(path=None):
+    """
+    Convert Yaml File to Dictionary
+
+    Args:
+        path (str): path to yaml file
+
+    Returns:
+        res (dic): return dictionary [yaml source]
+
+    """
+    full_path = os.path.join(Path(__file__).parent, path)
+    with open(full_path, "r") as file:
+        res = yaml.load(file.read(), Loader=yaml.Loader) or {}
+        if not isinstance(res, dict):
+            raise ValueError("Invalid yaml file")
+        return res
+
+
+def dict_to_yaml(path, data):
+    """
+    Convert Dictionary to Yaml file.
+
+    Args:
+        path (str): path to yaml file
+        data (dic): data of yaml
+
+    Returns:
+        path (dic): return path to yaml file
+
+    """
+    path = os.path.join(path, "build_config.yaml")
+    with open(path, "w+") as outfile:
+        yaml.dump(data, outfile, default_flow_style=False)
+    return path
+
+
+ceph_key_out = send_cmd(
+    "oc rsh --kubeconfig /opt/cluster/p1/auth/kubeconfig -n openshift-storage $(oc get --kubeconfig "
+    "/opt/cluster/p1/auth/kubeconfig pods -o wide -n openshift-storage|grep tool|awk '{print$1}') "
+    "cat /etc/ceph/keyring |grep key"
+)
+ceph_key_ls = ceph_key_out.split(" ")
+ceph_key = ceph_key_ls[2].replace("\n", "")
+
+endpoint_out = send_cmd(
+    "oc get --kubeconfig /opt/cluster/p1/auth/kubeconfig storagecluster -o yaml"
+    " -n openshift-storage|grep -i endpo"
+)
+endpoint_ls = endpoint_out.split(" ")
+endpoint = endpoint_ls[5].replace("\n", "")
```

Comment:
This should confirm somebody from MS team if it is the correct way to get the endpoint, because my understanding was, that this should be changed to getting IP from one `worker` node which is not `infra` node. (see also this [bug](https://url.corp.redhat.com/f80d64d))

And also it might be nicer to parse the output as `yaml`, instead of using `grep`, `split` etc.